config/rewrites.d/support.rackspace.com.json

```diff
@@ -191,5 +191,12 @@
         "rewrite": false,
         "status": 301
       }
```

Comment:
@DonNateR  Don't forget to add a comma between each rule. Missing one on line 193.